UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/MetaDataNode.cs

```diff
@@ -32,7 +32,7 @@ public class MetaDataNode : IGasMixContainer
 	/// <summary>
 	/// Used for calculating explosion data
 	/// </summary>
-	public ExplosionNode ExplosionNode = null;
+	public List<ExplosionNode> ExplosionNodes = new List<ExplosionNode>();
```

Comment:
Might just be  better to just make two separate nodes? 
or 
roll it into a node with, reference to 2 versions

Reply:
why 2 nodes? there should be possibility to add as much node types as you want, not only emps and regular explosions

Reply:
using array now